include/dlaf/blas/tile.h

```diff
@@ -81,6 +81,23 @@ inline void extendROCBlasWorkspace(cublasHandle_t handle,
   template <typename T>               \
   struct Name
 
+#ifdef DLAF_WITH_HIP
+    // rocblas functions needs to be lowercase
+#define DLAF_MAKE_GPUBLAS_OP(Name, f)                      \
+  DLAF_DECLARE_GPUBLAS_OP(Name);                           \
+  DLAF_DEFINE_GPUBLAS_OP(Name, float, s##f);               \
+  DLAF_DEFINE_GPUBLAS_OP(Name, double, d##f);              \
+  DLAF_DEFINE_GPUBLAS_OP(Name, std::complex<float>, c##f); \
+  DLAF_DEFINE_GPUBLAS_OP(Name, std::complex<double>, z##f)
+
+#define DLAF_MAKE_GPUBLAS_SYHE_OP(Name, f)                   \
+  DLAF_DECLARE_GPUBLAS_OP(Name);                             \
+  DLAF_DEFINE_GPUBLAS_OP(Name, float, Ssy##f);               \
+  DLAF_DEFINE_GPUBLAS_OP(Name, double, Dsy##f);              \
+  DLAF_DEFINE_GPUBLAS_OP(Name, std::complex<float>, Che##f); \
+  DLAF_DEFINE_GPUBLAS_OP(Name, std::complex<double>, Zhe##f)
```

Comment:
```suggestion
  DLAF_DEFINE_GPUBLAS_OP(Name, float, ssy##f);               \
  DLAF_DEFINE_GPUBLAS_OP(Name, double, dsy##f);              \
  DLAF_DEFINE_GPUBLAS_OP(Name, std::complex<float>, che##f); \
  DLAF_DEFINE_GPUBLAS_OP(Name, std::complex<double>, zhe##f)
```
or is this what your last `TODO` commit is about...?

Reply:
Unpushed?